src/app/components/color_picker.tsx

```diff
@@ -441,19 +441,21 @@ export class GradientView extends React.PureComponent<
       this.props.gradient.colors,
       this.props.gradient.colorspace
     );
-    const data = ctx.getImageData(0, 0, width, height);
-    for (let i = 0; i < data.width; i++) {
-      const t = i / (data.width - 1);
-      const c = scale(t);
-      for (let y = 0; y < data.height; y++) {
-        let ptr = (i + y * data.width) * 4;
-        data.data[ptr++] = c.r;
-        data.data[ptr++] = c.g;
-        data.data[ptr++] = c.b;
-        data.data[ptr++] = 255;
+    setImmediate(() => {
```

Comment:
Looks like this is not well supported - https://caniuse.com/#search=setImmediate.  Do we have a shim being loaded for this?

Reply:
Good catch! I'll try setTimeout(..., 0) instead. It doesn't have to be immediate.